src/clarity-icons/clarity-icons-element.ts

```diff
@@ -36,52 +33,49 @@ ClarityIconElement.prototype.constructor = ClarityIconElement;
 
 let generateIcon =
     function (element: any, shape: string) {
+        let cIcons = ClarityIconsApi.instance;
```

Comment:
I think it's better to set `cIcons` variable outside of `generateIcon` function unless there is a good reason to have it inside here. Otherwise, currently, we are accessing the common `ClarityIconsApi.instance` and setting `cIcons ` to it on every single icon.

Can we also rename this `cIcons` variable? As it's specifically referring to the `ClarityIconsApi.instance`, could we perhaps use `clrIconsApi` or something like that 😛 ?